recipes/tefinder/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "tefinder" %}
+{% set version = "2.32" %}
+
+package :
+  name: {{ name }}
+  version: {{ version }} 
+
+build:
+  number: 0
+  run_exports: 
```

yamllint

[error] trailing spaces

(trailing-spaces)